### PR TITLE
[SUSI] Change title and description for DESCRIPTION field

### PR DESCRIPTION
**Pull request by WDFYvonne**

### CODE REVIEW

**Comment by wurzka** [comment]
Can you please also update the schema?

File: file-formats/susi/type/zif_aff_susi_v1.intf.abap [hunk 1]
```diff
@@ -81,8 +81,8 @@ INTERFACE zif_aff_susi_v1
       "! $default {@link zif_aff_susi_v1.data:co_maintenance_mode.manual}
       "! $showAlways
       maintenance_mode TYPE ty_maintenance_mode,
-      "! <p class="shorttext">Description</p>
-      "! Description of authorization default values
+      "! <p class="shorttext">Documentation</p>
+      "! Authorization Default Documentation for Application
       description      TYPE string,
```
[the remark this answers is not in the document]
Comment (wurzka) [reply file-formats/susi/type/zif_aff_susi_v1.intf.abap:86]: Technically, this change would be incompatible. But as far as I know, the SUSI file format is not yet in use. So we could do this change.

[the remark this answers is not in the document]
Comment (WDFYvonne) [reply file-formats/susi/type/zif_aff_susi_v1.intf.abap:86]: SUSI is use for WEBI. If we change the DESCRIPTION field to DOCUMENTATION we also need to change it in the handler classes

[the remark this answers is not in the document]
Comment (WDFYvonne) [reply file-formats/susi/type/zif_aff_susi_v1.intf.abap]: I have adjusted the field in aff and handler casses

File: file-formats/susi/type/zif_aff_susi_v1.intf.abap [hunk 2]
```diff
@@ -129,8 +129,8 @@ INTERFACE zif_aff_susi_v1
       "! $default {@link zif_aff_susi_v1.data:co_maintenance_status.default_with_values}
       "! $showAlways
       maintenance_status TYPE ty_maintenance_status,
-      "! <p class="shorttext">Description</p>
-      "! Description of authorization defaults of object
+      "! <p class="shorttext">Documentation</p>
+      "! Authorization Default Documentation for Object
       description        TYPE string,
```
[the remark this answers is not in the document]
Comment (WDFYvonne) [reply file-formats/susi/type/zif_aff_susi_v1.intf.abap]: adjusted

**Comment by schneidermic0** [comment]
As far as I understood, you discussed this. This (incompatible) change is still possible because the ABAP file format is not used in any repository like abapGit, yet.

Correct me if my assumption is wrong.

**Review by wurzka — APPROVED** [review]
Looks good to me

**Pull request merged**